mainnet-1/specs/cosmoshub.json

```diff
@@ -0,0 +1,242 @@
+{
+    "proposal": {
+        "title": "Add Specs: Cosmos hub",
+        "description": "Adding new specification support for relaying cosmoshub data on Lava Mainnet",
+        "specs": [
+            {
+                "index": "COSMOSHUB",
+                "name": "cosmos hub mainnet",
+                "enabled": true,
+                "imports": [
+                    "COSMOSSDKFULL",
+                    "COSMOSSDK45DEP"
```

Comment:
this inheritance is outdated as cosmos hub is already on v50

Reply:
OK